src/app/admin/searchquiz/components/searchresult.tsx

```diff
@@ -0,0 +1,427 @@
+      "Test your knowledge of Python programming fundamentals with this quiz.",
+  },
+  {
+    _id: "65720501096a3c2ae50c6885",
+    title: "Python Fundamentals Quiz 2",
+    topic: "Python",
+    tags: ["Python", "Programming", "Fundamentals"],
+    coverImage: "https://npauldev.in/ProfilePictureHD.959b411b.webp",
+    description:
+      "Test your knowledge of Python programming fundamentals with this quiz.",
+  },
+  {
+    _id: "6572052eb45ee36e6c79817a",
+    title: "Python Fundamentals Quiz 2",
+    topic: "Python",
+    tags: ["Python", "Programming", "Fundamentals"],
+    coverImage: "https://npauldev.in/ProfilePictureHD.959b411b.webp",
+    description:
+      "Test your knowledge of Python programming fundamentals with this quiz.",
+  },
+  {
+    _id: "657206b888d86a55d22f4e19",
+    title: "Python Fundamentals Quiz 2",
+    topic: "Python",
+    tags: ["Python", "Programming", "Fundamentals"],
+    coverImage: "https://npauldev.in/ProfilePictureHD.959b411b.webp",
+    description:
+      "Test your knowledge of Python programming fundamentals with this quiz.",
+  },
+  {
+    _id: "657207320210ba3bf42c0e72",
+    title: "Python Fundamentals Quiz 2",
+    topic: "Python",
+    tags: ["Python", "Programming", "Fundamentals"],
+    coverImage: "https://npauldev.in/ProfilePictureHD.959b411b.webp",
+    description:
+      "Test your knowledge of Python programming fundamentals with this quiz.",
+  },
+  {
+    _id: "657209459be14cc4e0917920",
+    title: "Python Fundamentals Quiz 2",
+    topic: "Python",
+    tags: ["Python", "Programming", "Fundamentals"],
+    coverImage: "https://npauldev.in/ProfilePictureHD.959b411b.webp",
+    description:
+      "Test your knowledge of Python programming fundamentals with this quiz.",
+  },
+  {
+    _id: "657209e377a044426226ea82",
+    title: "Python Fundamentals Quiz 2",
+    topic: "Python",
+    tags: ["Python", "Programming", "Fundamentals"],
+    coverImage: "https://npauldev.in/ProfilePictureHD.959b411b.webp",
+    description:
+      "Test your knowledge of Python programming fundamentals with this quiz.",
+  },
+  {
+    _id: "65720f9f70c828b24100ace0",
+    title: "Python Fundamentals Quiz 2",
+    topic: "Python",
+    tags: ["Python", "Programming", "Fundamentals"],
+    coverImage: "https://npauldev.in/ProfilePictureHD.959b411b.webp",
+    description:
+      "Test your knowledge of Python programming fundamentals with this quiz.",
+  },
+  {
+    _id: "65720fdc70c828b24100ace7",
+    title: "Python Fundamentals Quiz 2",
+    topic: "Python",
+    tags: ["Python", "Programming", "Fundamentals"],
+    coverImage: "https://npauldev.in/ProfilePictureHD.959b411b.webp",
+    description:
+      "Test your knowledge of Python programming fundamentals with this quiz.",
+  },
+];
+
+export type QuizType = {
+  _id: string;
+  coverImage: string;
+  tags: string[];
+  title: string;
+  topic: string;
+  description: string;
+};
+
+export const columns: ColumnDef<QuizType>[] = [
```

Comment:
Keep this columns data in a seperate file called -> column.data.ts and import it in this file